ibis/backends/base/sql/alchemy/datatypes.py

```diff
@@ -9,6 +9,7 @@
 
 import ibis.expr.datatypes as dt
 from ibis.backends.base.sql.alchemy.geospatial import geospatial_supported
+from ibis.backends.base.sql.glot.datatypes import SqlglotType
```

Comment:
I really do not like this split naming pattern where anything starting with `sql` becomes `sql/$THING` perhaps this can break tradition and be `ibis.backends.base.sqlglot`?

Reply:
Agree, just wanted to align with the existing naming

Reply:
Done.